### DISCUSSION TITLE
maintaining ProjectChrono Package with dependencies

### DISCUSSION THREAD

**Discussion by DonBetur** [msys2/MINGW-packages#21848] [Ideas]
Hi everyone, i would like to suggest making a ProjectChrono package for mingw64-x86_64. 
Project Chrono is a powerfull open source multi-physics simulation engine, C++ api. 

Here it's repo: https://github.com/projectchrono/chrono
here it's website: https://projectchrono.org/

Most of packages are already present in list of packages provided by MSYS2, so, main question is, is it difficult to make this package available? 
Needed packages can be found in installation guides for each modules, i suggest making a full installation package, or some minimum for running a decent simulations. Minimum modules are: Core, Irrlicht (or OpenGL), Postprocess. 
So to sum it up...
presented packages are: 
mingw-w64-glfw, 
mingw-w64-glew,
 mingw-w64-glm, 
mingw-w64-opencascade, 
mingw-w64-swig, 
mingw-w64-irrlicht, 
mingw-w64-spectra,
mingw-w64-mumps,
(gnuplot?... povray, blender, )
puthon,
openMPI (gcc, g++?..)
absent packages in MSYS: blaze, thrust (CUDA includes), OptiX, TensoRT, openCRG, CRGterrain, VSG (visualization tool), 
and this hard ones, such as: pardiso MKL, CUDA, Matlab(if installed, specify path)

There are several versions of Project Chrono, here are the installation guides:
https://api.projectchrono.org/9.0.0/
https://api.projectchrono.org/8.0.0/

please share your opinion on this one who is interested in making this possible for LTS.

**Comment by raedrizqie** [comment]
MKL and CUDA is a blocker on mingw..

**Reply by DonBetur** [reply]
In that case, maybe without them?

**Reply by raedrizqie** [reply]
If you can make it work, why not? :)
